src/orders/views/OrderDetails/OrderDetailsMessages.tsx

```diff
@@ -337,6 +345,22 @@ export const OrderDetailsMessages: React.FC<OrderDetailsMessages> = ({
     }
   };
 
+  const handleTransactionAction = (
+    data: OrderTransactionRequestActionMutation,
+  ) => {
+    const {
+      transactionRequestAction: { errors },
+    } = data;
+    const isError = !!errors.length;
+
+    pushMessage({
+      status: isError ? "error" : "success",
+      text: isError
+        ? getOrderTransactionErrorMessage(errors[0], intl)
```

Comment:
If multiple errors happen do we care only about the first?

Reply:
yes, that's how we handle most errors in notifications 😅 (at least the ones I've checked)